packages/react/src/components/Forms/LabelledBox/tokens.ts

```diff
@@ -0,0 +1,8 @@
+import { DefaultTokens } from "@openfun/cunningham-tokens";
+
+export const tokens = (defaults: DefaultTokens) => ({
+  "label-color--small": defaults.theme.colors["greyscale-600"],
+  "label-color--big": defaults.theme.colors["greyscale-900"],
```

Comment:
is it normal that big label get a different color ?

Reply:
yep